src/reducers/githubReducer.js

```diff
@@ -1,14 +1,16 @@
 import initialState from './initialState';
-import { GITHUB_FETCH_TOTAL_ISSUES, GITHUB_FETCH_OPEN_ISSUES } from '../actions/githubActions';
+import { GITHUB_FETCH_TOTAL_ISSUES, GITHUB_FETCH_OPEN_ISSUES, GITHUB_FETCH_RELEASES } from '../actions/githubActions';
 
 export default function github(state = initialState.github, action) {
   const { payload } = action;
 
   switch (action.type) {
```

Comment:
personal opinion case statements make me sad, you can do 

```
return {
    GITHUB_FETCH_TOTAL_ISSUES: Object.assign({}, state, { totalIssues: payload }),
    GITHUB_FETCH_OPEN_ISSUES: Object.assign({}, state, { openIssues: payload })
}[action.type]
```

Reply:
```
return {
    GITHUB_FETCH_TOTAL_ISSUES: Object.assign({}, state, { totalIssues: payload }),
    GITHUB_FETCH_OPEN_ISSUES: Object.assign({}, state, { openIssues: payload })
}[action.type] || state;
```

I guess you could do this with the default?